src/Maestro/Maestro.Web/Helpers.cs

```diff
@@ -99,5 +101,60 @@ public static async Task<IActionResult> ProxyRequestAsync(this HttpContext conte
             }
 
         }
+
+        public static Dictionary<string, KustoQuery> CreateBuildTimesQueries(string repository, string branch, int days)
+        {
+            var parameters = new List<KustoParameter> {
+                new KustoParameter("_Repository", KustoDataTypes.String,  repository.Split("/").Last()),
+                new KustoParameter("_SourceBranch", KustoDataTypes.String, branch),
+                new KustoParameter("_Days", KustoDataTypes.TimeSpan, $"{days}d")
+            };
+
+            string publicProject = "public";
+
+            // Builds in AzDo are only found in the internal project
+            if (repository.Contains("dev.azure.com"))
+            {
+                publicProject = "internal";
+            }
+
+            // We only care about builds that complete successfully or partially successfully 
+            // from the given repository. We summarize duration of the builds over the last 7 days.
```

Comment:
>7 [](start = 92, length = 1)

nit: query is based on the parameter days, not just 7.

Reply:
Yes. Will update the comment